### ISSUE TITLE
Separate data generation components to a separate library?

### ISSUE THREAD

**Issue by jenniferjiangkells** [dotimplement/HealthChain#60]
Relatively modular, makes library bloated. Revive DoppelData + CDA / FHIR parsers?
Consider also separate library for visualisation components (mock EHR interface)

**Comment by jenniferjiangkells** [comment]
@adamkells you could also look at this

**Comment by adamkells** [comment]
Yeah I agree. It's a bit orthogonal to the rest of the repo. I think this can be its own thing. I'll look into separating this out and creating a roadmap.

**Comment by jenniferjiangkells** [comment]
@adamkells did we complete this?

**Comment by adamkells** [comment]
@jenniferjiangkells No we looked into it but it was slightly awkward as some components of the data generation code were used in other places. I can revisit this now that the interoperability engine is complete and it might be easier to remove this.

**Comment by jenniferjiangkells** [comment]
@adamkells ah ok. wondering if we could simplify it and roll it in with #114
